docs/tutorials/work_with_lsdb.ipynb

```diff
@@ -0,0 +1,236 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    %pip install aiohttp lsdb
</pre><p>Just a note, I'm suggesting we add dask-expr as a dependency to this workflow, but it's already a requirement of the package so it's not needed here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-dask/pull/18/discussion/#comment-1613809674'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>